docs/source/installation.rst

```diff
@@ -1,210 +1,159 @@
 Installation
 ============
 
-Supported OS
-++++++++++++
-    
-    ``ivadomed`` officially supports GPU acceleration only on ``Linux`` and CPU on ``Linux``, 
-    ``Windows`` and ``MacOS``.
+At your command prompt, run:
 
-.. _installation_step1:
- 
-Step 1: Set up a dedicated virtual environment
-++++++++++++++++++++++++++++++++++++++++++++++
+::
 
-    You can set up a virtual environment for ``ivadomed`` using either conda or venv:
+    pip install ivadomed
 
-    .. tabs::
+This should work with most combinations of OSes and pythons. If not, come ask us for help.
 
-        .. tab:: Install via ``venv``
+.. TODO: is this note even worth having, given that 99% of people are running Ubuntu which is configured correctly?
+.. note::
 
-            1. Set up Python Venv Virtual Environment.
+    If you are installing into your home folder (``pip install --user``), which is the default,
+    you need to make sure that ``~/.local/bin/`` (or ``%APPDATA%\Python\PythonXY\Scripts`` on Windows)
+    is on your ``$PATH`` for ivadomed's tools to be available. ``pip`` will warn you if it is not.
+    It should be pre-configured properly on Ubuntu and most common Linux distros.
 
-                ``ivadomed`` requires Python >= 3.7 and <3.10. First, make sure that a
-                compatible version of Python 3 is installed on your system by running:
+    If you are installing into a `venv <https://docs.python.org/3/library/venv.html>`_ or a
+    `conda env <https://docs.conda.io/projects/conda/en/latest/commands.html#conda-vs-pip-vs-virtualenv-commands>`_
+    then the tools will be available whenever that environment is activated.
```

Comment:
Maybe this section would be better in a separate Troubleshooting section. 80% of people will be fine, it's only people on like, ArchLinux or Gentoo or Alpine that might have this problem, and it's a one-time fix for them too so they might already have fixed it before they try to install us.